Mollie.WebApplicationCoreExample/Models/DecimalPlacesAttribute.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System.ComponentModel.DataAnnotations;
+using System.Globalization;
+
+namespace Mollie.WebApplicationCoreExample.Models
+{
+    public class DecimalPlacesAttribute : ValidationAttribute
+    {
+        public int DecimalPlaces { get; }
+
+        public DecimalPlacesAttribute(int decimalPlaces)
+        {
+            DecimalPlaces = decimalPlaces;
+        }
+
+        protected override ValidationResult IsValid(object value, ValidationContext validationContext)
+        {
+            decimal amount = (decimal)value;
+            string text = amount.ToString(CultureInfo.InvariantCulture);
+            int dotIndex = text.IndexOf('.');
+            var decimals = text.Length - dotIndex - 1;
+            return dotIndex < 0 || dotIndex != text.LastIndexOf('.') || decimals != DecimalPlaces
+                ? new ValidationResult(ErrorMessage ?? $"Please enter an amount with ${DecimalPlaces} decimal place(s)")
```

Comment:
I smell JavaScript.

Reply:
Lol, where do you smell JavaScript?

Reply:
The `$"Please enter an amount with ${DecimalPlaces} decimal place(s)"` part. Interpolated strings in C# do not use the $ sign for the expressions inside the string. C# only uses the $ in front to identify the string as an interpolated string. JavaScript uses the $ for every expression inside the template string. When i'm coding backend in C# and front-end in JavaScript I always mix those two up myself. 😄

Reply:
Yes! I always make this mistake because I'm doing so much Typescript programming lately.  Fixing it ASAP. Well spotted!

Reply:
See new commit. Tried to implement your feedback. 

I also activated C# 8 language features in the example project. Could do without of course.